src/java/picard/sam/ValidateSamFile.java

```diff
@@ -51,7 +51,15 @@
  * @author Doug Voet
  */
 @CommandLineProgramProperties(
-        usage = "Read a SAM or BAM file and report on its validity.",
+        usage = "Read a SAM or BAM file and report on its validity." +
```

Comment:
Would be helpful to note that by default, the tool will exit after finding 100 (I think -- should check the default number, it's an option) errors and output them to the console (stdout). I find it more useful to start by running in SUMMARY mode, which writes a summary to a file (I think). Also useful to note that you can specify types of errors to ignore, because some errors are not important.